ui/apps/platform/src/reducers/pageSearch.ts

```diff
@@ -1,150 +0,0 @@
-            options,
-        }),
-        [`set${capitalize(prefix)}SearchModifiers`]: (modifiers: SearchEntry[]) => ({
-            type: `${prefix}/SET_SEARCH_MODIFIERS`,
-            modifiers,
-        }),
-        [`set${capitalize(prefix)}SearchSuggestions`]: (suggestions: SearchEntry[]) => ({
-            type: `${prefix}/SET_SEARCH_SUGGESTIONS`,
-            suggestions,
-        }),
-    } as SearchActions<EntityPrefix>;
-}
-
-// Reducers
-
-type OneOfSearchActions<T extends string> =
-    | SetSearchOptionsAction<T>
-    | SetSearchModifiersAction<T>
-    | SetSearchSuggestionsAction<T>;
-
-type SearchReducer<T extends string> = (
-    state: SearchEntry[],
-    action: OneOfSearchActions<T>
-) => SearchEntry[];
-
-type Reducers<T extends string> = {
-    searchOptions: SearchReducer<T>;
-    searchModifiers: SearchReducer<T>;
-    searchSuggestions: SearchReducer<T>;
-};
-
-export function reducers<EntityPrefix extends string>(
-    prefix: EntityPrefix
-): Reducers<EntityPrefix> {
-    const searchOptions: SearchReducer<EntityPrefix> = (state = [], action) => {
-        if (action.type === `${prefix}/SET_SEARCH_OPTIONS`) {
-            const { options } = action as SetSearchOptionsAction<EntityPrefix>;
-            return isEqual(options, state) ? state : options;
-        }
-        return state;
-    };
-    const searchModifiers: SearchReducer<EntityPrefix> = (state = [], action) => {
-        if (action.type === `${prefix}/SET_SEARCH_MODIFIERS`) {
-            const { modifiers } = action as SetSearchModifiersAction<EntityPrefix>;
-            return isEqual(modifiers, state) ? state : modifiers;
-        }
-        return state;
-    };
-    const searchSuggestions: SearchReducer<EntityPrefix> = (state = [], action) => {
-        if (action.type === `${prefix}/SET_SEARCH_SUGGESTIONS`) {
-            const { suggestions } = action as SetSearchSuggestionsAction<EntityPrefix>;
-            return isEqual(suggestions, state) ? state : suggestions;
-        }
-        return state;
-    };
-    return {
-        searchOptions,
-        searchModifiers,
-        searchSuggestions,
-    };
-}
-
-// Selectors
-
-type SearchSelectorNames<T extends string> =
-    | `get${Capitalize<T>}SearchOptions`
-    | `get${Capitalize<T>}SearchModifiers`
-    | `get${Capitalize<T>}SearchSuggestions`;
-
-type SearchSelectors<T extends string> = {
-    [prop in SearchSelectorNames<T>]: (state: SearchState) => SearchEntry[];
-};
-
-export function getSelectors<EntityPrefix extends string>(
-    prefix: EntityPrefix
-): SearchSelectors<EntityPrefix> {
-    return {
-        [`get${capitalize(prefix)}SearchOptions`]: (state: SearchState) => state.searchOptions,
-        [`get${capitalize(prefix)}SearchModifiers`]: (state: SearchState) => state.searchModifiers,
-        [`get${capitalize(prefix)}SearchSuggestions`]: (state: SearchState) =>
-            state.searchSuggestions,
-    } as SearchSelectors<EntityPrefix>;
-}
```

Comment:
oh was this the last instance of using the page search reducers?

Reply:
Yes, no more page search reducers.

On the other side of the coin, `URLSearchInput` still uses searchAutocomplete reducer and sagas.